ed25519jni/.cargo/config

```diff
@@ -0,0 +1,2 @@
+[build]
+target-dir = "target/rust"
```

Comment:
I'm surprised we needed to introduce a `.cargo/config`. I'm assuming this is so the subcrate shares the build directory with the parent crate. I think we might be able to avoid this by adding `ed25519jni` as a workspace member of the parent crate.

Reply:
Sure, I'll give it a try. I'm not yet familiar with all the Rust ins & outs. :)